lib/ramble/docs/tutorials/modifying_a_gromacs_experiment.rst

```diff
@@ -0,0 +1,221 @@
+.. Copyright 2022-2023 Google LLC
+
+   Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+   https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+   <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+   option. This file may not be copied, modified, or distributed
+   except according to those terms.
+
+.. _modifying_an_experiment_tutorial:
+
+=======================================
+3) Modifying A GROMACS Experiment
+=======================================
+
+In this tutorial, you will learn how to modify an existing workspace
+configuration that contains experiments using
+`GROMACS <https://www.gromacs.org/>`_, a free and open-source application
+for molecular dynamics.
+
+This tutorial builds off of concepts introduced in previous tutorials. Please
+make sure you review those before starting with this tutorial's content.
+
+.. include:: shared/gromacs_workspace.rst
+
+Activate the Workspace
+----------------------
+
+As you are using a pre-existing workspace, ensure it is activated (NOTE: you
+only need to run this if you do not currently have the workspace active).
+
+.. code-block:: console
+
+    $ ramble workspace activate basic_gromacs
+
+
+Experiment Descriptions
+-----------------------
+
+Now that your workspace has been configured, and activated, You can execute the
+following command to see what experiments the workspace currently contains:
+
+.. code-block:: console
+
+    $ ramble workspace info
+
+This command provides a summary view of the workspace. It includes the
+experiment names, and the software environments. As an example, its output
+might contain the following information:
+
+.. code-block:: console
+
+    Experiments:
+      Application: gromacs
+        Workload: water_gmx50
+          Experiment: gromacs.water_gmx50.pme_single_rank
+      Application: gromacs
+        Workload: water_gmx50
+          Experiment: gromacs.water_gmx50.rf_single_rank
+      Application: gromacs
+        Workload: water_bare
+          Experiment: gromacs.water_bare.pme_single_rank
+      Application: gromacs
+        Workload: water_bare
+          Experiment: gromacs.water_bare.rf_single_rank
+
+To get detailed information about where variable definitions come from, you can use:
+
+.. code-block:: console
+
+    $ ramble workspace info -v
+
+The experiments section of this command's output might contain the following:
+
+.. code-block:: console
+
+    Experiments:
```

Comment:
Why is this different from the output in tutorial 2?

Reply:
One uses `ramble workspace info` (To show the experimens ) and the other uses `ramble info`  (to show the application info)